request_handler.py

```diff
@@ -144,12 +155,11 @@ def do_DELETE(self):
             delete_user(id)
         if resource == "comments":
             delete_comment(id)
+            self.wfile.write(f"{delete_comment}".encode())
```

Comment:
this line should be taken out. Same example as the post method, you should refactor that line to scoped for the entire method like like 152 instead of scoped for the conditional.

Reply:
If I understand your suggestion correctly, I think I have fixed it